.github/workflows/remoteconfig.yml

```diff
@@ -90,6 +90,34 @@ jobs:
     - name: Test Swift Quickstart
       run: ([ -z $plist_secret ] || scripts/third_party/travis/retry.sh scripts/test_quickstart.sh Config)
 
+  quickstart_release_test:
```

Comment:
I'm not sure about putting this in remoteconfig.yml versus a separate release.yml for all of them.

Reply:
I prefer a separate release.yml since 
1. badges on readme will be very specific to the latest pods in repo. The result is intuitive to developers.
2. If release testings need to be triggered, we don't need to trigger nightly quickstart testing.